Src/Base/AMReX_GpuLaunch.H

```diff
@@ -215,6 +231,22 @@ namespace Gpu {
 #define AMREX_HOST_DEVICE_PARALLEL_FOR_3D(...) AMREX_GPU_HOST_DEVICE_PARALLEL_FOR_3D(__VA_ARGS__)
 #define AMREX_HOST_DEVICE_PARALLEL_FOR_4D(...) AMREX_GPU_HOST_DEVICE_PARALLEL_FOR_4D(__VA_ARGS__)
 
+#ifdef AMREX_USE_CUDA
+#define AMREX_HOST_DEVICE_PARALLEL_FOR_1D_FUSABLE(...) AMREX_GPU_HOST_DEVICE_FOR_1D_FUSABLE(__VA_ARGS__)
```

Comment:
I think this spelling is a bit confusing. This brings to mind the pronunciation "fussy" instead of "fuse".

 I'd lean towards "FUSEABLE" to be very clear, or, apparently the proper English seems to be "FUSIBLE": https://wikidiff.com/fusable/fusible.

Reply:
OK.  Then let's use the proper English work.